doc/secciones/06_analisis.tex

```diff
@@ -1,2 +1,203 @@
-\chapter{Análisis del problema}
- 
+\chapter{Análisis}
+
+En este capítulo, se realizará una explicación breve de la metodología de trabajo seguida en el mismo y un análisis de requisitos. Posteriormente, se procederá al proceso de selección lenguajes que más tarde se empleará para empezar a diseñar los objetos de nuestro proyecto.
+
+\section{Domain-Driven Design}
+
+En este capítulo, se hará un análisis detallado de los requisitos del proyecto haciendo uso del Diseño Orientado al Dominio o \textit{domain-driven design} que para mayor rapidez y concisión denominaremos como \textit{DDD}. Esta filosofía de desarrollo de software nos ayudará a comprender y modelar el dominio empresarial, asegurando que nuestras soluciones se alineen con las necesidades específicas de los usuarios y del negocio.
+
+El Diseño Orientado al Dominio (DDD) es una filosofía de desarrollo de software que se centra en comprender y modelar el dominio empresarial para alinear el software con las necesidades del negocio. Introducido por Eric Evans en su libro~\cite{evans2004domain}, DDD ha evolucionado con aportes de numerosos profesionales.
+
+En esencia, \textit{DDD} aborda la complejidad enfocándose en el ``dominio'', el contexto específico del software. Promueve el uso de un ``lenguaje ubicuo'', un lenguaje común entre desarrolladores y partes interesadas, para garantizar que el software refleje con precisión el ámbito empresarial. La modelización en \textit{DDD} no busca crear el modelo más realista, sino uno útil y apropiado para su propósito, similar a cómo una película representa la realidad de manera selectiva para cumplir su objetivo.
```

Comment:
Conviene que estas cosas las cuentes, pero también las uses. El "lenguaje ubicuo" tendrás que mencionarlo en los mensajes de commit o de alguna forma cuando en los issues abordes el modelado de alguna parte.
En general, conviene que tratéis de entender lo que estáis poniendo y cómo os ayuda a resolver el problema, y al contrario, si algo no lo entendéis, no lo pongáis o preguntéis.

Reply:
He añadido una parte para el mismo en 9fe3a4d además se estará utilizando en el proyecto.